rumbo_logic/src/jobs/mongo.rs

```diff
@@ -0,0 +1,60 @@
+pub use super::prelude::*;
+use mongodb::Collection;
+
+pub struct MongoJobStorageService {
+    db_adapter: Arc<DbAdapter>,
+}
+
+impl MongoJobStorageService {
+    pub fn new(db: &Arc<DbAdapter>) -> Self {
+        MongoJobStorageService {
+            db_adapter: db.clone(),
+        }
+    }
+
+    pub fn as_arc(self) -> Arc<Self> {
+        Arc::from(self)
+    }
+
+    fn get_collection(&self) -> Collection<JobInfo> {
+        static COLLECTION_NAME: &str = "JobInfo";
+        self.db_adapter.get_collection(COLLECTION_NAME)
+    }
+}
+
+#[async_trait]
+impl JobStorageService for MongoJobStorageService {
+    async fn save(&self, info: JobInfo) -> Result<Option<JobInfo>> {
+        let collection = self.get_collection();
+
+        let id = &info.name.clone();
+
+        let some = collection
+            .replace_one(get_id_filter(&info), info, None)
+            .await?;
+
+        if some.modified_count == 0 {
+            return Err(RumboError::MongoError("Updated count = 0".to_string()));
```

Comment:
Maybe we need to store all immutable string variable in constants? I think it`s more efficient.